7.9.0 release notes (#59961)
elastic/elasticsearch: docs/java-rest/low-level/configuration.asciidoc

include-tagged::{doc-tests}/RestClientDocumentation.java[rest-client-config-timeouts]
--------------------------------------------------

> Timeouts also can set per request with RequestOptions. This will override RestClient customizeRequestConfig.

Reviewer: Not sure what this is doing in with the release notes, but it looks like it's missing a word.
```suggestion
Timeouts also can be set per request with RequestOptions, which overrides RestClient customizeRequestConfig.
```

Author: Ah, I meant to mention that - in tracking down what one of the PRs was for, I found a change to the documentation that needed some improvement (but didn't improve enough 😁).
